Checker.cpp

```diff
@@ -2,21 +2,36 @@
 #include <iostream>
 using namespace std;
 
+bool isWithinRange(float value, float min, float max, std::string Message) {
+    if (value < min || value > max) {
```

Comment:
decouple cout statement from this function . Wrap in another function

Reply:
resolved